man/man1/viz_pipe.1

```diff
@@ -0,0 +1,14 @@
+./" Man page for viz_pipe
+.TH man 1 "26 November 2020" "1.0" "viz_pipe man page"
+.SH NAME
+viz_pipe -/ Generate pipeline visual
```

Comment:
I think we should use this instead: "Generates a network visualization from a configuration file."